kamailio_icscf/icscf.cfg

```diff
@@ -13,6 +13,7 @@ alias=ims.mnc001.mcc001.3gppnetwork.org
 #!define HOSTNAME "icscf.ims.mnc001.mcc001.3gppnetwork.org"
 
 #!subst "/NETWORKNAME/ims.mnc001.mcc001.3gppnetwork.org/"
+#!subst "/HSS_REALM/epc.mnc001.mcc001.3gppnetwork.org/"
```

Comment:
please remove the changes related to HSS_REALM. I would like to keep the changes which correspond to only to regex to eliminate ck/ik parameters in WWW-Authenticate

Reply:
The rational behind this change is that the original tutorial to create an IMS system was based on a separate HSS for the IMS part, but with the latest open5gs code, it is no longer necessary to use a separate HSS: the HSS code of the open5gs project is sufficient. Hence, since the open5gs HSS is on the epc side, its realm will be epc... rather than ims...
If you still want to keep only the regex change, I will remove the HSS_REALM commit.

Reply:
I am aware that the open5gs HSS supports Cx interface. However, this repo is used as part of a tutorial in open5gs involving usage of FHoSS for Cx interface. So it would be great if you could just retain regex change and remove other changes. Many thanks for the extra effort.

Reply:
ok, but what about this: keeping HSS_REALM but give it default value ims... so that your script are still compatible with FHoHSS but become easily adaptable to open5gs HSS?